samples/XCT.Sample/Pages/Views/SnackBarPage.xaml.cs

```diff
@@ -151,6 +151,36 @@ async void DisplaySnackBarAdvancedClicked(object? sender, EventArgs args)
 			StatusText.Text = result ? "SnackBar is closed by user" : "SnackBar is closed by timeout";
 		}
 
+		async void Issue1883(object sender, EventArgs args)
```

Comment:
Could you create this repo in [this folder](https://github.com/xamarin/XamarinCommunityToolkit/tree/main/samples/XCT.Sample/Pages/TestCases)? We use it for our bug reports